### PR TITLE
Fix release workflow

### PR DESCRIPTION
**Pull request by sjspielman**
I'm filing this PR to fix the failure of the release GHA which runs on pulls (noting the job was misnamed as `renv`, but this is indeed the release GHA): https://github.com/AlexsLemonade/rOpenScPCA/actions/runs/11805537264/job/32888217441.

Opening as a draft to get this passing before requesting review.

### CODE REVIEW

**Comment by jashapiro** [comment]
looks like renv is overriding the dependency installation: you need to remove the renv file before installing dependencies & running, as here: https://github.com/AlexsLemonade/scpcaTools/blob/e1c1e2f934c77c693e4bd50675d46cb64c8b8736/.github/workflows/R-CMD-check-current.yaml#L27-L28

File: .github/workflows/R-CMD-CHECK-release.yml [hunk 1]
```diff
@@ -5,11 +5,12 @@ on:
     branches:
       - main
       - feature/*
+  pull_request:
```
Comment (jashapiro) [inline .github/workflows/R-CMD-CHECK-release.yml:9]: If this is generally working, I think we can get rid of this trigger agian.
```suggestion

```

**Review by jashapiro — APPROVED** [review]
LGTM

File: .github/workflows/R-CMD-CHECK-release.yml (hunk 1 above)
Comment (sjspielman) [reply .github/workflows/R-CMD-CHECK-release.yml:9]: ah! yes. slightly gas-addled brain over here 🥴

**Pull request merged**